### PR TITLE
fix: Disable the "USE AS ALIAS" button when the user does not have a profile

### PR DESCRIPTION
**Pull request by cyaiox**
This PR disables the `Use as Alias button` when the user doesn't have a profile and shows a tooltip.

Screen Shot 2022-07-13 at 17 22 45

Closes: #2125

### CODE REVIEW

File: src/modules/translation/languages/en.json [hunk 1]
```diff
@@ -1502,4 +1503,4 @@
       "subtitle": "Thank you for revieweing this collection"
     }
   }
-}
+}
```
Comment (juanmahidalgo) [inline src/modules/translation/languages/en.json:1506]: mhmm, is your IDE removing the extra newline in here? I think this change shouldn't be here 😅

Comment (juanmahidalgo) [reply src/modules/translation/languages/en.json:1506]: same for other `.json` files!

File: src/components/ENSListPage/ENSListPage.tsx [hunk 2]
```diff
@@ -89,8 +89,18 @@ export default class ENSListPage extends React.PureComponent<Props, State> {
...
+        <Button className="ui basic button" onClick={() => this.handleOpenModal(ens.name)} disabled={!hasProfileCreated}>
+          {t('ens_list_page.button.use_as_alias')}
+        </Button>
+      </div>
+    )
+  }
+
   renderEnsList() {
-    const { ensList } = this.props
+    const { ensList, hasProfileCreated } = this.props
```
Comment (juanmahidalgo) [inline src/components/ENSListPage/ENSListPage.tsx:103]:
```suggestion
    const { ensList, avatar } = this.props
    const hasProfileCreated = !!avatar
```
what do you think about this alternative? so we keep the prop as the selector

File: src/modules/translation/languages/en.json (hunk 1 above)
Comment (cyaiox) [reply src/modules/translation/languages/en.json]: Thanks for the review! Updated

**Review by juanmahidalgo — APPROVED** [review]
LGTM 🚀

**Pull request merged**